ValetTouchIDTest/ValetTouchIDTestAppDelegate.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ValetTouchIDTestAppDelegate.swift
+//  Valet
+//
+//  Created by Eric Muller on 4/20/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//
+
+import UIKit
+
+
+@UIApplicationMain
+
+
+final class ValetTouchIDTestAppDelegate : UIResponder, UIApplicationDelegate
+{
+    var window: UIWindow?
+}
```

Comment:
missing trailing newline (do we care about this in Swift? I can be swayed either way here)